app/scripts/controllers/transactions/index.test.js

```diff
@@ -1195,6 +1284,22 @@ describe('Transaction Controller', function () {
 
       txController.cancelTransaction(0);
     });
+
+    it('should reject the approval request', function () {
+      txController.cancelTransaction(0);
+
+      assert.equal(messengerMock.call.callCount, 1);
+      assert.deepEqual(messengerMock.call.getCall(0).args, [
+        'ApprovalController:rejectRequest',
+        '0',
+        new Error('Rejected'),
+      ]);
+    });
+
+    it('should not throw if rejecting approval request throws', async function () {
```

Comment:
From a look at the `ControllerMessenger` code, calling an action can fail when:
- No action is passed.
- Action is not in the allow list
- Action is not registered

Given that we are making these calls internally, we should not see it failing but... wouldn't it be best to let it fail so that we can catch any typos or miss-configurations during development?